lib/active_merchant/billing/gateways/beanstream/ipp_core.rb

```diff
@@ -151,23 +160,23 @@ def new_submit_xml(action)
         xml.target!
       end
 
-      def commit_url
-        (test? ? test_url : live_url)
+      def ipp_commit_url
+        (test? ? IPP_TEST_URL : IPP_LIVE_URL)
```

Comment:
please split the commits into two 
1) renaming (`_ipp`, `remote_beanstream_ipp_test.rb`...)
2) code changes
thx